src/lib.rs

```diff
@@ -481,7 +492,7 @@ impl From<Vec<u8>> for Der<'static> {
 
 impl fmt::Debug for Der<'_> {
     fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
-        f.debug_tuple("Der").field(&self.as_ref()).finish()
+        hex(f, self.as_ref())
```

Comment:
Did you consider leaving the `Der(..)` wrapping in? Why/why not?

Reply:
I left this unchanged, both options seem reasonable to me.

Reply:
It winds up with multiple layers of wrapping in the Debug output, which gets kind of noisy. For instance, since `CertificateDer` is:

```rust
#[derive(Clone, Debug, PartialEq, Eq)]
pub struct CertificateDer<'a>(Der<'a>);
```

It would output as `CertificateDer ( Der ( 03040506 ) )`. To most readers the inner `Der` wrapping is an implementation detail and not that important. But I don't feel strongly about it.

Reply:
Fair enough!